pyflp/plugin.py

```diff
@@ -113,6 +131,10 @@ class FruityBalanceEvent(StructEventBase):
     STRUCT = _FruityBalanceStruct
 
 
+class FruityBloodOverdriveEvent(StructEventBase):
+    STRUCT = _FruityBloodOverdriveStruct
+
+
```

Comment:
The BO event would set both versions of the struct, so NativePluginEvent can use the one it decides to be appropriate.
```
class FruityBloodOverdriveEvent(NativePluginEvent)
    struct_old = _FruityBloodOverdriveOldStruct
    struct_new = _FruityBloodOverdriveNewStruct

```

Reply:
NativePluginEvent would look at the events stream and decide which plugin version it encountered.

```
class NativePluginEvent(StructEventBase)

    # check the version of the plugin based on stream size and/or plugin marker in the first 4 bytes?
   
    # set the appropriate struct

```

Reply:
I need to consider whether I should just modify existing `StructEventBase` logic to accomodate multiple structs, since its very possible that the functionality `NativePluginEvent` has might be used by other events